resteasy-client-microprofile-base/src/main/java/org/jboss/resteasy/microprofile/client/RestClientBuilderImpl.java

```diff
@@ -451,9 +466,6 @@ private boolean isMapperDisabled() {
                 // ignore cast exception
             }
         }
-        if (disabled) {
-            LOGGER.warnf("The default ResponseExceptionMapper has been disabled");
-        }
```

Comment:
This warning is not really nice as we override the default ResponseExceptionMapper. I think it's a perfectly valid use case and it shouldn't raise a warning.
Worst case, we can silence it with some logging config on our side.